output/schema/schema.json

```diff
@@ -91495,12 +91492,9 @@
         },
         {
           "availability": {
-            "serverless": {
-              "stability": "experimental"
-            },
+            "serverless": {},
             "stack": {
-              "since": "8.15.0",
-              "stability": "experimental"
+              "since": "8.15.0"
```

Comment:
Did you automatically regenerate these files, or hand edit them? I think we still need the stack version information and I'm surprised if it got wiped out automatically.

Reply:
It depends. If the API did not change since 8.15, then it's fine, otherwise we should switch to 8.19.

Reply:
(Also, sorry for the 2 weeks delay! Asking devtools-team for a review can help with faster review time, as there are many changes here owned by the docs team which we don't review.)

Reply:
> It depends. If the API did not change since 8.15, then it's fine, otherwise we should switch to 8.19.

I think we're good unless there's a change I don't know about (@kderusso ?)